apps/condo/domains/user/schema/User.js

```diff
@@ -130,6 +131,30 @@ const User = new GQLListSchema('User', {
         },
 
     },
+    hooks: {
+        afterChange: async ({ updatedItem, context, existingItem, operation }) => {
+            if (
```

Comment:
it's better to group this code by creating a function `updateEmployeesRelatedToUser`